src/caduceus/http.go

```diff
@@ -118,6 +120,7 @@ type ProfileHandler struct {
 // that the main handler has successfully dealt with
 func (ph *ProfileHandler) ServeHTTP(response http.ResponseWriter, request *http.Request) {
 	logging.Info(ph.Logger).Log(logging.MessageKey(), "Receiving request for server stats...")
```

Comment:
This is way too much logging.  Remove this entirely.  If we want request logs, that should be handled in middleware.